runtime/src/genesis_utils.rs

```diff
@@ -245,20 +244,14 @@ pub fn create_genesis_config_with_alpenglow_vote_accounts_no_program(
     voting_keypairs: &[impl Borrow<ValidatorVoteKeypairs>],
     stakes: Vec<u64>,
 ) -> GenesisConfigInfo {
-    create_genesis_config_with_alpenglow_vote_accounts(
```

Comment:
We can probably remove `create_genesis_config_with_alpenglow_vote_accounts_no_program` now? Just have everyone switch to `create_genesis_config_with_alpenglow_vote_accounts`